cmake/CMakeLists.download_gtest.in

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.6)
```

Comment:
the cmake version is too high, the minimum version brpc supports now is version 2.8.10, above which cmake is widely used. And why is this value? The command you use in this file is all supported by 2.8.10

Reply:
Yes, this is an oversight when I brought this cmake gudget from one of my own repos. I will fix it ASAP.